pkg/virt-operator/install-strategy/aggregatorclient.go

```diff
@@ -0,0 +1,89 @@
+package installstrategy
```

Comment:
how is `apiregistration.go`? My intent was to use apiVersion name instead of the "Kind" for the file names, which clearly got mixed with the client name instead at some point... This wont track for scc however since "security.go" is to vague, as, I believe, is "monitoring.go" for prometheus resources.

Reply:
If you want to go with function instead of kind I would suggest `apiserver(s).go`, since that is what we actually do: registering an additional apiserver.